Gemfile

```diff
@@ -51,13 +51,25 @@ gem "bootsnap", require: false
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem "image_processing", "~> 1.2"
 
+gem "devise"
+
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'dotenv-rails'
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'guard-rspec'
+  gem 'rspec-rails'
+  gem 'rubocop-rails'
+  gem 'rubocop-rspec'
```

Comment:
I think you need to add this gem to the .rubocop.yml to make it work, Same for rubocop-rails

Reply:
right 

```yml
require:
  - rubocop-rspec
```